src/components/dashboard.jsx

```diff
@@ -12,12 +12,24 @@ import CurrentUserStore from '../user-store';
 import AsyncButton from './async-button';
 import Time from './time';
 
-const SAMPLE_REPOS = [
+let SAMPLE_REPOS = [
```

Comment:
You don't need to change this to `let` since you still could do `push`. Also, uppercase words in a variable name must be use `const`.